### ISSUE TITLE
Cannot access workflow license save button via keyboard only

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#18915]
the tab target moves from 'license' to 'a person'
![image](https://github.com/user-attachments/assets/6a802b1f-b65a-40da-85a0-2af90820602c)

tested on usegalaxy.eu running 24.1
